index.js

```diff
@@ -0,0 +1,141 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
```

Comment:
I'm not sure if there's a performance effect on this, but I guess it's better to only use the typescript parser on .ts(x) files (you can set it in the override object) and use the default eslint parser for .js(x) files.

Reply:
If we do that, we also must move the TypeScript specific `parserOptions`.

Reply:
Yeah, `project: './tsconfig.json',` should be moved as well.